docs/cdk-erigon/deploy-testnet.md

```diff
@@ -1,208 +1,1007 @@
+# Set this if cdk-erigon will be the sequencer
+export CDK_ERIGON_SEQUENCER=1
+
+# Path to your cdk-erigon binary and config file
+/path/to/cdk-erigon/build/bin/cdk-erigon \
+  --config=/path/to/my-cdk-chain-config/dynamic-{yournetwork}.yaml \
+  --zkevm.genesis-config-path=/path/to/my-cdk-chain-config/ # Directory of allocs, chainspec etc.
+  # Add other CLI flags as needed (e.g., --datadir if not in YAML)
 ```
-
-## Monitoring and Debugging
-
-### View Service Logs
-
-The following commands work on both macOS and Linux:
-
+*   `cdk-erigon` will initialize its data directory and start syncing/sequencing. Monitor its logs.
+
+### 3.5. (For Validium) Running the Data Availability Committee (DAC) Nodes
+
+If you're setting up a Validium, you need to run Data Availability Committee (DAC) nodes. These nodes store the L2 transaction data off-L1 and provide attestations.
+
+*   **Get DAC Node Software**:
+    *   The DAC node software is typically found in a repository like `0xPolygon/cdk-data-availability`.
+    ```bash
+    git clone https://github.com/0xPolygon/cdk-data-availability.git
+    cd cdk-data-availability
+    ```
+*   **Build DAC Nodes**:
+    *   Follow the build instructions in the repository (likely Go-based).
+*   **Configure DAC Nodes**:
+    *   Each DAC node will require configuration, including:
+        *   Its own private key for signing attestations.
+        *   Connection details for other DAC nodes (for P2P communication).
+        *   Connection to the L1 `DataCommittee.sol` contract to know the current committee members and rules.
+        *   Storage backend for the L2 data.
+        *   Network ports for listening to data from `cdk-node` and for inter-DAC communication.
+    *   *Refer to the `0xPolygon/cdk-data-availability` repository for example configurations and startup scripts.*
+*   **Run DAC Nodes**:
+    *   Start multiple DAC nodes (as per your committee size and fault tolerance requirements).
+    *   Ensure they can communicate with each other and with your `cdk-node` (which will send data to them).
+
+### 4. Running `cdk-node`
 ```bash
-# View aggregation layer logs
-kurtosis service logs cdk agglayer --follow
-
-# View sequencer logs
-kurtosis service logs cdk cdk-erigon-sequencer-001 --follow
+# Path to your cdk-node binary and its config.toml
+/path/to/cdk-node/binary \
+  --config /path/to/my-cdk-chain-config/cdk-node-config.toml
+  # Add other CLI flags as needed
 ```
+*   `cdk-node` will connect to `cdk-erigon` and Sepolia. 
+*   If `cdk-erigon` is the main sequencer (`CDK_ERIGON_SEQUENCER=1` set for it), `cdk-node` acts as a **Sequence Sender**, taking blocks produced by `cdk-erigon` and posting them to L1 (or DAC).
+*   If `cdk-erigon` is *not* the sequencer, `cdk-node` itself will sequence transactions from the L2 tx pool.
+*   It will start processing L2 transactions, forming batches, and sending them to L1 (or to DAC for Validium). Monitor its logs.
```

Comment:
I'm not sure this is accurate, there are only two sequencers for these networks, zkevm-node which is deprecated now and doesn't support the latest forks, and cdk-erigon.  Let's remove these lines.